Userland/Libraries/LibGfx/JPGLoader.cpp

```diff
@@ -447,35 +447,29 @@ static inline bool is_valid_marker(const Marker marker)
     return false;
 }
 
-static inline ErrorOr<u16> read_be_word(InputMemoryStream& stream)
+static inline ErrorOr<u16> read_be_word(Core::Stream::Stream& stream)
 {
-    BigEndian<u16> tmp;
-    stream >> tmp;
-    TRY(stream.try_handle_any_error());
-    return tmp;
+    return AK::convert_between_host_and_big_endian(TRY(stream.read_value<u16>()));
```

Comment:
Is it possible to get `TRY(stream.read_value<BigEndia.<U16>>())` to work? That'd be a bit more expressive.